package.json

```
@@ -1,34 +1,54 @@
 {
   "name": "todolist",
-  "version": "0.2.0",
+  "version": "1.0.0",
   "private": true,
+  "type": "module",
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "default": "./dist/index.js"
+    }
+  },
+  "types": "dist/index.d.ts",
   "scripts": {
     "build": "tsc",
-    "start": "ts-node src/todolist.ts",
-    "start:watch": "node -r ts-node/register --watch src/todolist.ts",
-    "test": "jest"
+    "start": "node --loader=ts-node/esm src/todolist.ts",
+    "start:watch": "node --loader=ts-node/esm --watch src/todolist.ts",
+    "test": "NODE_OPTIONS=\"--experimental-vm-modules\" jest"
   },
   "dependencies": {
     "cors": "^2.8.5",
-    "express": "^4.17.3",
+    "express": "^4.18.2",
```

Why don't use the latest version ?
```suggestion
    "express": "^4.19.2",
```